packages/compass-saved-aggregations-queries/src/components/saved-item-card.tsx

```diff
@@ -18,16 +18,16 @@ import {
 } from '@mongodb-js/compass-components';
 import { formatDate } from '../utlis/format-date';
 
-type Action = 'open' | 'delete' | 'copy' | 'edit';
+export type Action = 'open' | 'delete' | 'copy' | 'rename';
 
-type SavedItemCardProps = {
+export type SavedItemCardProps = {
```

Comment:
suggestion: should we extend `Item` from store?

Reply:
Done in ab45f29e5e9b7be9c0165340a3da5ed0f1d6dc0d. Used a `Pick` so that we are explicit about exactly which parts of the model should be part of the UI state